code/_onclick/hud/movable_screen_objects.dm

```diff
@@ -21,6 +21,7 @@
 
 
 /obj/screen/movable/MouseDrop(over_object, src_location, over_location, src_control, over_control, params)
+	. = ..()
```

Comment:
Won't this cause the obj/screens to be passed into whatever things they're over. I mean the things should still be typechecked but still.